### PR TITLE
[AAP-21400] Constructed inventories tech debt work 

### PR DESCRIPTION
**Pull request by fullsushidev**
relates to [AAP-21400](https://issues.redhat.com/browse/AAP-21400)

### CODE REVIEW

File: cypress/e2e/awx/resources/inventoriesConstructed.cy.ts [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      });
+  });
+
+  it.skip('can edit the input_inventories, verify the preservation of the order they were added in, and manually change the order', () => {
+    //Create a constructed inventory in the beforeEach hook
+    //Assert the original order of the input inventories
+    //Assert the UI change to the order of input inventories
+  });
+
+  it.skip('shows a failed sync on the constructed inventory if the user sets strict to true and enters bad variables', () => {
```
Comment (vidyanambiar) [inline cypress/e2e/awx/resources/inventoriesConstructed.cy.ts:175]: Do we need a follow-up JIRA for these?

File: cypress/e2e/awx/resources/inventoriesConstructed.cy.ts [hunk 2]
```diff
@@ -0,0 +1,197 @@
...
+    cy.intercept({
+      method: 'GET',
+      pathname: awxAPI`/inventories/`,
+      query: { name__icontains: invNames[invToCreate - 1] },
+    }).as('filterInputInventories');
+    cy.multiSelectBy('[data-cy="inventories"]', invNames);
+    cy.wait('@filterInputInventories');
+    cy.getByDataCy('update_cache_timeout').clear().type(String(cacheTimeoutValue));
+    cy.singleSelectByDataCy('verbosity', String(verbosityValue));
+    // FIXME: when verbosity is 0 the filed remains empty
```
Comment (vidyanambiar) [inline cypress/e2e/awx/resources/inventoriesConstructed.cy.ts:97]: Is this comment required?

File: cypress/e2e/awx/resources/inventoriesConstructed.cy.ts [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+    cy.clickButton(/^Save inventory$/);
+    cy.wait('@saveInv')
+      .its('response.statusCode')
+      .then((statusCode) => {
+        expect(statusCode).to.be.equal(200);
+        cy.verifyPageTitle(constructedInv.name);
+        cy.getByDataCy('description').contains(description);
+      });
+    cy.clickButton(/^Sync inventory$/);
+    // let jobID: number;
```
Comment (vidyanambiar) [inline cypress/e2e/awx/resources/inventoriesConstructed.cy.ts:156]: Can we get rid of this comment?

**Review by vidyanambiar — APPROVED** [review]
lgtm!

Added a couple of minor comments.

File: cypress/e2e/awx/resources/inventoriesConstructed.cy.ts (hunk 2 above)
Comment (fullsushidev) [reply cypress/e2e/awx/resources/inventoriesConstructed.cy.ts:97]: The issue was fixed. I'm removing the comment now.

File: cypress/e2e/awx/resources/inventoriesConstructed.cy.ts (hunk 1 above)
Comment (fullsushidev) [reply cypress/e2e/awx/resources/inventoriesConstructed.cy.ts:175]: no, it will be addressed in the next PR

**Review by akus062381 — APPROVED** [review]
Looks good :-)

**Pull request merged**